client/shipping-label/state/reducer.js

```diff
@@ -422,21 +422,24 @@ reducers[ SET_PACKAGE_TYPE ] = ( state, { packageId, boxTypeId } ) => {
 	const newPackages = { ...state.form.packages.selected };
 	const oldPackage = newPackages[ packageId ];
 
+	const oldBox = state.form.packages.all[ oldPackage.box_id ];
+	const newBox = state.form.packages.all[ boxTypeId ];
+	const weight =
```

Comment:
Fun fact! This isn't a JS bug. As funny as it is to see `0.1 + 0.2 !== 0.3` this is part of the IEEE standard for the number format. Check out much more detail at this post: https://medium.com/the-node-js-collection/javascripts-number-type-8d59199db1b6